alerta/auth/oidc.py

```diff
@@ -18,7 +18,7 @@
 def get_oidc_configuration(app):
 
     OIDC_ISSUER_URL_BY_PROVIDER = {
-        'azure': 'https://sts.windows.net/{}/'.format(app.config['AZURE_TENANT']),
+        'azure': 'https://login.microsoftonline.com//{}/v2.0'.format(app.config['AZURE_TENANT']),
```

Comment:
There's an extra slash between `.com` and the tenant id which is preventing the issuer URL from matching.